engine/src/test/java/io/camunda/zeebe/process/test/engine/EngineClientTest.java

```diff
@@ -332,6 +333,78 @@ void shouldCancelProcessInstance() {
             });
   }
 
+  @Test
+  void shouldModifyProcessInstance() throws InterruptedException, TimeoutException {
+    // given
+    zeebeClient
+        .newDeployResourceCommand()
+        .addProcessModel(
+            Bpmn.createExecutableProcess("simpleProcess")
+                .startEvent()
+                .serviceTask("task1", task -> task.zeebeJobType("task1"))
+                .serviceTask("task2", task -> task.zeebeJobType("task2"))
+                .endEvent()
+                .done(),
+            "simpleProcess.bpmn")
+        .send()
+        .join();
+
+    final ProcessInstanceEvent processInstance =
+        zeebeClient
+            .newCreateInstanceCommand()
+            .bpmnProcessId("simpleProcess")
+            .latestVersion()
+            .send()
+            .join();
+
+    zeebeEngine.waitForIdleState(Duration.ofSeconds(1));
+
+    final long task1Key =
+        StreamSupport.stream(
+                RecordStream.of(zeebeEngine.getRecordStreamSource())
+                    .processInstanceRecords()
+                    .spliterator(),
+                false)
+            .filter(r -> r.getIntent() == ProcessInstanceIntent.ACTIVATE_ELEMENT)
```

Comment:
❓ Could it be that this was a typo and instead of `ACTIVATE_ELEMENT` it was intended to find the first `ELEMENT_ACTIVATED`?

Reply:
You are correct. It shouldn't matter functionality-wise, but I'll change it either way. Good catch!